### DISCUSSION TITLE
import files from custom folder

### DISCUSSION THREAD

**Discussion by riemsesy** [sircharlo/meeting-media-manager#376] [Ideas] [answered]
Hello Charles,
Is it an idea to also be able to point to a folder location to import files. The location can contain multiple pictures and videos and all media files in the folder will be imported. 

**Or** to select multiple files to import.

Thanks for all your work, we are enjoying the program very much!

**Comment by sircharlo** [comment, marked as answer]
Hi there!
When importing custom files, you can already import multiple files.

### Multiple file selection

![image](https://user-images.githubusercontent.com/1275666/171019837-b404d052-2541-4760-b016-2b9848123d13.png)

![image](https://user-images.githubusercontent.com/1275666/171019872-4cb03e9c-4a6b-4aee-8b11-3417fa4933f2.png)


### Drag-and-drop

You can also drap-and-drop files onto the window once "custom" is selected:

![image](https://user-images.githubusercontent.com/1275666/171020095-a3810fb0-7679-46f6-b66c-fa2b8af64014.png)

![image](https://user-images.githubusercontent.com/1275666/171020143-4421c638-def7-4404-8a86-3c1be7951fa0.png)

**Reply by riemsesy** [reply]
Great!  Why didn't I think of trying that.
